packages/vite-node/src/cli.ts

```diff
@@ -35,10 +45,13 @@ export interface CliOptions {
   mode?: string
   watch?: boolean
   options?: ViteNodeServerOptionsCLI
+  version?: boolean
+  help?: boolean
   '--'?: string[]
 }
 
 async function run(files: string[], options: CliOptions = {}) {
+  // console.log('run', files, options)
```

Comment:
Looks like you left a comment here

Reply:
Sorry, I forgot to remove the debug code, I'll fix it later